.github/workflows/ci.yml

```diff
@@ -0,0 +1,25 @@
+name: CI
+
+on: [push, pull_request]
+
+env:
+  FORCE_COLOR: 1
+jobs:
+  test:
+    strategy:
+      matrix:
+        python-version: ["3.10-dev", "3.9", "3.8", "3.7", "3.6"]
+        os: [macos-latest, windows-latest, ubuntu-latest]
+    runs-on: ${{ matrix.os }}
+    env:
+      IS_COVERAGE_ALLOWED: ${{ secrets.CODACY_PROJECT_TOKEN != '' }}
+      IS_MASTER_BRANCH: ${{ github.ref == 'refs/heads/master' }}
+    steps:
+    - uses: actions/checkout@master
+    - uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install pip dependencies
+      run: pip install -r requirements-test.txt
+    - name: Test
+      run: pytest -v test.py
```

Comment:
IMO it would be better to add a `test` extra and do `pip install .[test]`.

Reply:
What do you mean?